.generator/src/generator/templates/api.j2

```diff
@@ -0,0 +1,185 @@
+{% include "api_info.j2" %}
+
+
+from {{ config["packageName"] }}.api_client import ApiClient, Endpoint as _Endpoint
+from {{ config["packageName"] }}.model_utils import (
+    date,
+    datetime,
+    file_type,
+    none_type,
+)
+{%- for model in get_api_models(operations) %}
+from {{ config["packageName"] }}.model.{{ model|snake_case }} import {{ model }}
+{%- endfor %}
+
+{% set classname = name.replace(" ", "") + "Api" %}
+class {{ classname }}:
+
```

Comment:
can we add a tag description as docstring here?

Reply:
I added it in the init method, is that good enough?